### PR TITLE
Bump AGGrid version

### LINKED ISSUE #11650: Wrong data appears in table Viz
https://github.com/user-attachments/assets/d0139d6d-14a9-41a4-8fe1-6ee585595c7d

[Workflow.zip](https://github.com/user-attachments/files/17902074/Workflow.zip)

Version 2024.5.1-nightly.2024.11.25
Build e5a65a26cb521a124b047a7fd32f7abd01c0ab7c
Electron 31.2.0
Chrome 126.0.6478.127
User Agent Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) enso/2024.5.1-nightly.2024.11.25 Chrome/126.0.6478.127 Electron/31.2.0 Safari/537.36

### PR DESCRIPTION
**Pull request by farmaazon**
### Pull Request Description

Fixes #11650 - this was AGGrid issue, fixed in new version.

The bumped version also changes the menu of table column:

Screenshot From 2024-11-27 13-33-13

The bump also has shown problems in AGGridTableView component:
* The "grid" class inherits `display: grid` probably because of tailwind - fixed by renaming
* The classes inside AGGridTableView are not loaded in TableInputWidget - seems to be bug in Vue - if component is both used as normal and custom element, the styles seems to be applied to the latter. Worked around by defining classes higher, soon I will try to make simple reproduction and file an issue to Vue.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- ~~[ ] The documentation has been updated, if necessary.~~
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- ~~[ ] Unit tests have been written where possible.~~
- ~~[ ] If meaningful changes were made to logic or tests affecting Enso Cloud integration in the libraries, 
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.~~
  - If applicable, it is suggested to paste a link to a successful run of the Extra Tests.

### CODE REVIEW

File: app/gui/src/project-view/components/GraphEditor/GraphVisualization/visualizationData.ts [hunk 1]
```diff
@@ -159,6 +159,7 @@ export function useVisualizationData({
 
   const effectiveVisualizationData = computed(() => {
     const dataSourceValue = toValue(dataSource)
+    console.log('UPDATE VISUALIZATION DATA', dataSource)
```
Comment (somebody1234) [inline app/gui/src/project-view/components/GraphEditor/GraphVisualization/visualizationData.ts]: stray log?

**Review by kazcw — APPROVED** [review]
Looks good except for various extra `console.log` and `console.error`.

File: app/gui/playwright.config.ts [hunk 2]
```diff
@@ -128,19 +128,6 @@ export default defineConfig({
...
-      expect: {
-        toHaveScreenshot: { threshold: 0 },
-        timeout: TIMEOUT_MS,
-      },
-      timeout: TIMEOUT_MS,
-      use: {
-        baseURL: `http://localhost:${ports.dashboard}`,
-        actionTimeout: TIMEOUT_MS,
-      },
-    },
```
Comment (Frizi) [inline app/gui/playwright.config.ts:143]: Why is this removed?

**Review by Frizi — APPROVED** [review]

File: app/gui/playwright.config.ts (hunk 2 above)
Comment (farmaazon) [reply app/gui/playwright.config.ts:143]: There are no tests in this project. Agreed with @MrFlashAccount
